### PR TITLE
Extended team

### PR DESCRIPTION
**Pull request by ashtonmv**
This automatically pulls all github contributors onto the Team page via an ajax call to the github API. Related to #7.

The core developers still have special profiles on the Team page that also link to their MPIE pages. I'm not sure if this is how we want to have it, but it's easy to change if not.

Also, did we ever decide how the affiliated projects/external collaborators page should look? I didn't add anything about that because I'm not personally even sure what those projects/groups are.

### CODE REVIEW

**Comment by jan-janssen** [comment]
Hi @ashtonmv It took me some time to get everything setup on my local machine. I really like the extended team list. I was just wondering if it is possible to remove those who are already listed in the top row - mainly Joerg, Sudarsan, Liam, Sam and me from the lower part.

File: team.html [hunk 1]
```diff
@@ -50,8 +49,12 @@ <h2 style="padding-top: 40px;"><i class="fa fa-github-alt"></i></h2>
 	$.ajax(query).success(function (response) {
 		for (var i=0; i < response.length; i++) {
 			var username = response[i].login;
-			var bots = ['pyironlandingpage', 'codacy-badger', 'IceflowRE'];
-			if (!(jQuery.inArray(username, bots) > -1)) {
+			var excluded = [
+				'pyironlandingpage', 'codacy-badger', 'IceflowRE',
```
Comment (jan-janssen) [inline team.html:53]: Can we move this part to an external file or something? Just to make it easier for people to find it later on when the next person is updating the website.

Comment (ashtonmv) [reply team.html]: Yeah I think that's a good idea. In the last 2 commits I set it up to automatically exclude anyone with a markdown file under core_developers from the github contributors list, and then I listed all the bots in _config.yml under github_bots. I think this is the most lightweight/foolproof way to maintain this. Do you happen to know if the IceflowRE account is a real contributor or a bot?

**Comment by ashtonmv** [comment]
I think this one is ready to go live now that the steering committee and alumni have been added.

**Comment by jan-janssen** [comment]
Looks good to me

**Review by jan-janssen — APPROVED** [review]

**Pull request merged**